MediaBrowser.Common/Json/Converters/JsonInt32Converter.cs

```diff
@@ -0,0 +1,53 @@
+using System;
+using System.Buffers;
+using System.Buffers.Text;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace MediaBrowser.Common.Json.Converters
+{
+    /// <summary>
+    /// Converts a GUID object or value to/from JSON.
+    /// </summary>
+    public class JsonInt32Converter : JsonConverter<int>
+    {
+        /// <inheritdoc />
+        public override int Read(ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options)
+        {
+            static void ThrowFormatException() => throw new FormatException("Invalid format for an integer.");
```

Comment:
Do we need this as a defined function?

Reply:
We don’t need to... but it’s an optimization.

Reply:
Mmm you'd think it would be inlined anyway